### ISSUE TITLE
Deploy identity-slackbot

### ISSUE THREAD

**Issue by bramleyjl** [department-of-veterans-affairs/va.gov-team#77459]
# Issue Description

The [identity-slackbot](https://github.com/department-of-veterans-affairs/identity-slackbot) is currently hosted on my private Heroku account; several Identity engineers have access to it there.

<img width="1300" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0hhQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f08a0861fa2b8b53585b94dc04b93ed7cdaa301b/image.png" alt="image.png" />

This should be replaced by an official VA-owned deployment of the identity-slackbot.

# Tasks

- [x] Consult with platform support team engineers responsible for maintaining the VSP slackbot on their deployment setup
- ~~[x] determine if a VA Heroku account is a possibility~~
- [x] consult with @jhouston0 regarding a Kubernetes deployment
- [x] create a VA-owned identity-slackbot deployment

**Comment by jhouston0** [comment]
So, looking at the code, it is doable to deploy to Kubernetes. The issue is that this app needs an external endpoint, which @joeniquette has indicated is pretty difficult. The recommendation is either investigate going to socket mode in the code, or talk to the platform team and see what ideas they have

**Comment by bramleyjl** [comment]
The slackbot has been updated to work in socket mode, and the Heroku environment settings have been updated for the change.

We're currently waiting on the creation of the [ECR repo and relevant permissions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78596).

**Comment by bramleyjl** [comment]
Per @jhouston0 [CI/CD & container building are complete](https://dsva.slack.com/archives/C06FD8276DD/p1710953519581239?thread_ts=1710876135.254509&cid=C06FD8276DD), and he is now getting the slackbot app into our Argo.

**Comment by jhouston0** [comment]
PR is up for getting the slackbot into argo https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2774

**Comment by jhouston0** [comment]
the identity-slackbot is up in argo and appears to work. Waiting for a bug fix [PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2783) to be merged in and the deployment should be complete (its up now, but autosync might sync it at any time to the branch that has the bug)
<img width="1379" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2NQQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--216315e521327f14c6ef44f54fd4c3c6bdaac553/Screenshot%202024-03-26%20at%2011.14.38%E2%80%AFAM.png" alt="Screenshot 2024-03-26 at 11.14.38 AM.png" />

<img width="1064" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM29QQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1cd7d29839a9508e879548f0a994d3e34165ab14/Screenshot%202024-03-26%20at%2011.14.19%E2%80%AFAM.png" alt="Screenshot 2024-03-26 at 11.14.19 AM.png" />

**Comment by jhouston0** [comment]
PR is merged and slackbot is updated to the main branch. <img width="1051" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeGNXQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2df59a9f626948b9d070852b7b8e76643420f671/image.png" alt="image.png" />

**Comment by jhouston0** [comment]
identity slackbot CI/CD code has been merged into master, and updated argocd as expected
<img width="954" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0kxQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4d18bc1067fe7a1cf8aba39c85dfeba448d01f08/image.png" alt="image.png" />
<img width="1790" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL00xQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ab5e825a9900955c9070f109942ea933fdfeb064/image.png" alt="image.png" />

**Comment by bramleyjl** [comment]
I switched the Slack configuration over to socket mode and confirmed that it was interacting with the production Argo deploy.
